frontend/src/lib/Quiver.ts

```diff
@@ -37,9 +37,14 @@ interface TableCell {
   content: string
 }
 
-export interface Index {
+export interface Data {
```

Comment:
Would `TypedData` be an appropriate name here?

Reply:
The type names reflect Pandas DataFrame attributes.